Jenkinsfile.conda

```diff
@@ -0,0 +1,98 @@
+pipeline {
+    agent{ 
+        docker {
+            image 'lsstts/conda_package_builder:latest'
+            alwaysPull 'true'
+            args "--env TS_CONFIG_OCS_DIR=/home/saluser/ts_config_ocs --env LSST_DDS_DOMAIN=citest -u root --entrypoint=''"
+        }
+    }
+    environment {
+        package_name = "environment"
+        OSPL_HOME="/opt/OpenSpliceDDS/V6.9.0/HDE/x86_64.linux"
+    }
+    options {
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '14',
+                numToKeepStr: '10'
+            ))
+        disableConcurrentBuilds()
+    }
+    stages {
+        stage("Clone ts_environment"){
+            steps {
+                withEnv(["HOME=${env.WORKSPACE}"]) {
+                    sh """
+                    cd /home/saluser
+                    git clone https://github.com/lsst-ts/ts_config_ocs.git
+                    """
+                }
+            }
+        }
+        stage("Create Conda package") {
+            when {
+                buildingTag()
+            }
+            steps {
+                withEnv(["HOME=${env.WORKSPACE}"]) {
+                    sh """
+                    cd ${HOME}/conda 
+                    source /home/saluser/miniconda3/bin/activate
+                    conda config --add channels conda-forge
+                    conda config --add channels lsstts
+                    source ${OSPL_HOME}/release.com
+                    conda build --prefix-length 100 .
+                    """
+                }
+            }
+        }
+        stage("Create Conda Dev package") {
+            when {
+                not {
+                    buildingTag()
+                }
+            }
+            steps {
+                withEnv(["HOME=${env.WORKSPACE}"]) {
+                    sh """
+                        cd ${HOME}/conda
+                        source /home/saluser/miniconda3/bin/activate
+                        conda config --add channels conda-forge
+                        conda config --add channels lsstts
+                        source ${OSPL_HOME}/release.com
+                        conda build -c lsstts/label/dev --prefix-length 100 .
+                    """
+                }
+            }
+        }
+
+        stage("Push Conda Release package") {
+            when {
+                buildingTag()
+            }
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'CondaForge', passwordVariable: 'anaconda_pass', usernameVariable: 'anaconda_user')]) {
+                    withEnv(["HOME=${env.WORKSPACE}"]) {
+                        sh """
+                        source /home/saluser/miniconda3/bin/activate
+                        anaconda login --user ${anaconda_user} --password ${anaconda_pass}
+                        anaconda upload -u lsstts --force ~/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2
```

Comment:
I think this needs to be 

```
anaconda upload -u lsstts --force /home/saluser/miniconda3/conda-bld/linux-64/ts-${package_name}*.tar.bz2
```

Reply:
Using the full path name, instead of the shortcut.